kolibri/plugins/learn/assets/src/views/commonLearnStrings.js

```diff
@@ -76,6 +77,29 @@ export const learnStrings = createTranslator('CommonLearnStrings', {
     message: 'Explore',
     context: 'Resource and filter label for the type of learning activity. Translate as a VERB',
   },
+  dontShowThisAgainLabel: {
+    message: 'Don’t show this again',
+    context:
+      'Option that allows the user to prevent this resource from displaying in the future while using category search',
+  },
+  markResourceAsCompleteLabel: {
```

Comment:
This can also be reused in the [MarkAsCompleteModal](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/MarkAsCompleteModal.vue#L39).

Also, the `Resource completed` string appears in both [MarkAsCompleteModal](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/MarkAsCompleteModal.vue#L49) and the [CompletionModal](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/CompletionModal/index.vue#L356). Can you move it in the `commonLearn` strings and reuse?